articles/app-service-mobile/app-service-mobile-xamarin-forms-get-started-users.md

```diff
@@ -304,17 +342,19 @@ Universal Windows Platform (UWP) projects, but using the **UWP** project (with n
        {
```

Comment:
Delete the paragraph which begins with "2. (Optional) In the WinPhone81". It's not needed now.